src/Sylius/Bundle/ArchetypeBundle/Resources/meta/LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2011-2014 Paweł Jędrzejewski
```

Comment:
It's 2015 now unless you plan to update all the copyright years at the same time.